### DISCUSSION TITLE
https://warframe.market/   Does not Work on Thorium  (only problem i have so far)

### DISCUSSION THREAD

**Discussion by theogr91** [community/community#70815] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Sorry, you have been blocked
You are unable to access warframe.market

Why have I been blocked?
This website is using a security service to protect itself from online attacks. The action you just performed triggered the security solution. There are several actions that could trigger this block including submitting a certain word or phrase, a SQL command or malformed data.

What can I do to resolve this?
You can email the site owner to let them know you were blocked. Please include what you were doing when this page came up and the Cloudflare Ray ID found at the bottom of this page.
![Untitled](https://github.com/community/community/assets/22870410/5d756992-e352-48e6-b1d2-a8bd3f55ab50)

**Comment by queenofcorgis** [comment]
Hi @theogr91,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project or their community. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
